### ISSUE TITLE
MIT license is incompatible with the code which was copied from Flask-RESTful

### ISSUE THREAD

**Issue by joshfriend** [noirbizarre/flask-restplus#319]
I recognize a substantial amount of code from Flask-RESTful in this project (as noted originally in #149). Your inclusion of some attribution to the original authors added in 7edec90 helps, but you are still not able to re-license BSD code as MIT. Here's a helpful chart to illustrate license compatability:

![license compatibility](https://www.dwheeler.com/essays/floss-license-slide-image.png)

> An arrow from box A to box B means that you can combine software with these licenses; the combined result effectively has the license of B, possibly with additions from A.

([source](https://www.dwheeler.com/essays/floss-license-slide.html))

tl;dr: You need to change the license of this project to be BSD 3-clause
